### ISSUE TITLE
Importability for global variables in certain cases is significantly diminished

### ISSUE THREAD

**Issue by syntroniks** [joxeankoret/diaphora#180]
I could be insane, but...
https://github.com/joxeankoret/diaphora/blob/44dfc7d31401f94764f1f62f2de626527c7fb29c/diaphora_ida.py#L1279-L1280

In _my_ case, for _my_ database,
```if ea1 in matched_syms and ea2 in import_syms:
    self.import_instruction(matched_syms[ea1], import_syms[ea2])
```
works significantly better. Could this be a logic error? Surely other users would have come across this.

I am working with tricore databases where functions & global variables shift between target software revisions. With this change and #179, things are greatly improved.

**Comment by joxeankoret** [comment]
...and thanks for reporting this as well, as this looks to me like yet another bug.

**Comment by joxeankoret** [comment]
Sorry for taking so long. Should be fixed with this: b2a7918204e74c138d090eb42c315152278167da
